packages/server/src/fhir/operations/expand.test.ts

```diff
@@ -698,11 +698,18 @@ describe('Updated implementation', () => {
       .set('Authorization', 'Bearer ' + accessToken);
     expect(res.status).toEqual(200);
     const expansion = res.body.expansion as ValueSetExpansion;
+    const system = 'http://terminology.hl7.org/CodeSystem/v3-ActMood';
 
-    expect(expansion.contains).toHaveLength(11);
-    expect(expansion.contains).not.toContainEqual<ValueSetExpansionContains>({
-      code: '_ActMoodPredicate',
-    });
+    expect(expansion.contains).toHaveLength(5);
```

Comment:
This new state is actually more correct: the included pre-expanded ValueSet elides six deprecated/retired codes